how2meet/ui/components/event_editor.py

```diff
@@ -60,7 +82,7 @@ async def render(self, on_next, floating=True):
                     copy_icon._props["onclick"] = f"navigator.clipboard.writeText('{self.auth_token}');"
             elements.label("We can also send it to your email if you want.").classes("text-xl text-center w-full")
             with ui.row().classes("w-full items-center justify-between"):
-                self.email_input = elements.input("Email").classes("flex-grow")
+                self.email_input = elements.input("Email", validation=self.check_email).classes("flex-grow")
```

Comment:
So the validation arg here is what calls the email regex and displays red text if invalid?

Reply:
Yep. Validation takes a dict or a callable to make the decisions, but this is what causes the red error message I have shown in the screenshots